projects/plugins/jetpack/modules/search/instant-search/components/search-app.jsx

```diff
@@ -45,17 +47,35 @@ import './search-app.scss';
 
 class SearchApp extends Component {
 	static defaultProps = {
+		overlayOptions: {},
 		widgets: [],
 	};
 
 	constructor() {
 		super( ...arguments );
+
 		this.state = {
-			overlayOptions: { ...this.props.initialOverlayOptions },
 			isVisible: !! this.props.initialIsVisible, // initialIsVisible can be undefined
+			overlayOptionsCustomizerOverride: {},
 		};
+
 		this.getResults = debounce( this.getResults, 200 );
-		this.props.initializeQueryValues();
+		this.props.enableAnalytics && this.initializeAnalytics();
```

Comment:
`initializeAnalytics` is not called only when `enableAnalytics` is set to false, as expected . But unfortunately, the tracking is still enabled somehow - I can still see tracking data sent, e.g.
`https://pixel.wp.com/t.gif?fetch_algo=jetpack:search/1-score_default&fetch_position=2&fetch_query=hello&railcar=dmHMQ0jwdjjb&rec_blog_id=193141071&rec_post_id=44&session_id=TGN6e2&ui_algo=jetpack-instant-search-ui/v1&ui_position=2&_en=jetpack_instant_search_traintracks_render&_ui=Aywbd8KXVwkYLpcKh7BhG1O9&_ut=anon&_ts=1624492779267&_tz=-12&_lg=en-US&_pf=MacIntel&_ht=1120&_wd=1792&_sx=0&_sy=0&_dl=https://jasper.au.ngrok.io/?s=hello&_dr=&_rt=1624492779267&_=_`

Reply:
Drat -- I thought I sorted this out. I'll look into this.